### DISCUSSION TITLE
Windows support?

### DISCUSSION THREAD

**Discussion by Agrael1** [fair-acc/opencmw-cpp#291] [Ideas]
Hello, all the submodules are seemingly available for windows consumption with clang-cl at least.
Are there any plans to make it compilable on Windows?

**Comment by RalphSteinhagen** [comment]
Hi @Agrael1, thanks for your interest in our project. 

Yes, we want to make it accessible for Windows and Mac users too, but we lack expertise in OS-specific differences. Our team is mostly Linux developers, so we need help from experts in Windows and/or Mac OS. 

Please feel free to submit PRs and feedback if you'd like to help. :-)
